src/internal/utils/version.go

```diff
@@ -30,6 +31,69 @@ func IsValidOscalVersion(version string) error {
 	return nil
 }
 
+func validVersionFormat(version string) error {
+	if !versionRegexp.MatchString(version) {
+		return fmt.Errorf("version %s is not a valid version", version)
+	}
+	return nil
+}
+
+// GetLatestVersion returns the latest version of the OSCAL schema from the schemas directory
```

Comment:
Odd request - please feel free to say no. Could we add more comments here to reflect why this is a valuable implementation vs a Constant that is statically defined?